.gitignore

```diff
@@ -5,3 +5,5 @@ lib
 /npm-packages-offline-cache
 tmp
 yarn
+/package-lock.json
+/.vscode
```

Comment:
Ok I can see that for `.vscode`, but why not `package-lock.json` at this level?  I ask since `node_modules` are excluded in this file.